e2e-tests/tests/error-boundaries.test.ts

```diff
@@ -4,23 +4,65 @@
  *--------------------------------------------------------------------------------------------*/
 import { expect, Page, test } from "@playwright/test";
 import assert from "assert";
-import { tabLocator, widgetLocator } from "./Utils";
+import { popoutWidget, tabLocator, widgetLocator } from "./Utils";
 
-test.describe("error boundary", () => {
+async function testWidgetNotPoppedOut(page: Page) {
+  const tab = tabLocator(page, "Error border widget");
+  const widget = widgetLocator({ tab });
+
+  const button = widget.getByRole("button", { name: "Click Me" });
+  const error = widget.getByRole("alert");
+
+  await expect(error).not.toBeVisible();
+  await button.click();
+  await expect(error).toBeVisible();
+}
+
+async function testWidgetPoppedOut(page: Page) {
+  const tab = tabLocator(page, "Error border widget");
+  const widget = widgetLocator({ tab });
+
+  const popoutPage = await popoutWidget(widget);
+  const button = popoutPage.getByRole("button", { name: "Click Me" });
+  const error = popoutPage.getByRole("alert");
+
+  await expect(error).not.toBeVisible();
+  await button.click();
+  await expect(error).toBeVisible();
+}
+
+test.describe("error boundary when reparentPopoutWidgets is true", () => {
   test.beforeEach(async ({ page, baseURL }) => {
     assert(baseURL);
-    await page.goto(`${baseURL}/blank?frontstageId=widget-api`);
+    await page.goto(
+      `${baseURL}/blank?frontstageId=test-popout&reparentPopoutWidgets=1`
+    );
```

Comment:
This is consistent with other tests, however, we can avoid `assert` by using relative paths: https://playwright.dev/docs/test-webserver#adding-a-baseurl (I'd appreciate if you could make this change in all e2e tests)

Reply:
Updated the relative path in every test